app/models/application_record.rb

```diff
@@ -0,0 +1,5 @@
+require 'active_record'
```

Comment:
It ain't ringing bells, but I'm curious why the directory path includes ruby `2.3.0`. Is the correct ruby version loaded (2.3.4)?